sklearn/svm/tests/test_svm.py

```diff
@@ -230,22 +231,22 @@ def test_svr_errors():
     y = [0.0, 0.5]
 
     # Bad kernel
-    clf = svm.SVR(kernel=lambda x, y: np.array([[1.0]]))
+    clf = svm.SVR(gamma='scale', kernel=lambda x, y: np.array([[1.0]]))
     clf.fit(X, y)
     assert_raises(ValueError, clf.predict, X)
 
 
 def test_oneclass():
     # Test OneClassSVM
-    clf = svm.OneClassSVM()
+    clf = svm.OneClassSVM(gamma='scale')
```

Comment:
That should be gamma='auto' in order not to change the result, shouldn't it?

Reply:
> in order not to change the result

But I don't think that's the focus of this test is. It is to actually test the default `OneClassSVM`, which will eventually have (in version 0.22) `gamma='scale'` as default.

Infact at most of the places we have used `gamma='scale'` and not `gamma='auto'`.

Reply:
> Infact at most of the places we have used gamma='scale' and not gamma='auto'.

I saw that and I am not sure I understand why I have to admit ... is it to test `gamma='scale'` because it is the upcoming default, is it because at least docs should promote best practices, is it for some other reason that I am missing?

Reply:
The logic here is to add `gamma='scala'` in the tests now, and to remove them when default gamma is set to `scale` in the future. I agree with you that it might be better to use `gamma=auto` in this test to avoid modifying the result, thus to avoid protential test failures.
Since I don't think it will make too much difference, both solutions LGTM from my side.

Reply:
I think it is fine to leave it like this. At least the future default is well covered by tests.

Reply:
Fine, let's leave it like this :)